mixpanel/__init__.py

```diff
@@ -407,10 +407,18 @@ def send(self, endpoint, json_message):
         if endpoint not in self._buffers:
             raise MixpanelException('No such endpoint "{0}". Valid endpoints are one of {1}'.format(self._buffers.keys()))
 
-        buf = self._buffers[endpoint]
+        if endpoint != 'imports':
+            buf = self._buffers[endpoint]
+        elif endpoint == 'imports':
```

Comment:
I might want to spell this "else:" (since if endpoint != 'imports' implies endpoint == 'imports')